servers/rendering/renderer_rd/renderer_storage_rd.cpp

```diff
@@ -4825,6 +4825,7 @@ void RendererStorageRD::particles_request_process(RID p_particles) {
 
 	if (!particles->dirty) {
 		particles->dirty = true;
+		particles->inactive_time = 0;
```

Comment:
This change will just make it impossible for particles to become inactive as long as they are visible, so definitely not a correct fix. The main issue is that I just dont understand the problem and cant see anything in the reproduction project of the original issue.

Reply:
Hi,

Yea I definitely should have looked more into how that function was being called, I assumed that it was only being called when the a new particle was instanced and turns out, that's definitely wrong. The problem is, I am not seeing the behavior you're describing from this fix, the particle lasts only until its lifetime and disappears.

Any idea why this is happening?